client/controller/index.web.js

```diff
@@ -4,37 +4,59 @@
 import React from 'react';
 import ReactDom from 'react-dom';
 import { Provider as ReduxProvider } from 'react-redux';
-import { setSection as setSectionAction } from 'state/ui/actions';
-import noop from 'lodash/noop';
+import page from 'page';
 
 /**
- * Internal dependencies
+ * Internal Dependencies
  */
-import page from 'page';
-import LayoutLoggedOut from 'layout/logged-out';
+import Layout from 'layout';
+import layoutFocus from 'lib/layout-focus';
+import nuxWelcome from 'layout/nux-welcome';
+import translatorInvitation from 'layout/community-translator/invitation-utils';
+import userFactory from 'lib/user';
+import sitesFactory from 'lib/sites-list';
 import debugFactory from 'debug';
 
+const user = userFactory();
+const sites = sitesFactory();
 const debug = debugFactory( 'calypso:controller' );
 
-/**
- * @param { object } context -- Middleware context
- * @param { function } next -- Call next middleware in chain
- *
- * Produce a `LayoutLoggedOut` element in `context.layout`, using
- * `context.primary`, `context.secondary`, and `context.tertiary` to populate it.
-*/
-export function makeLoggedOutLayout( context, next ) {
+export { makeLoggedOutLayout } from './index.node.js';
+export { setSection } from './index.node.js';
```

Comment:
These two lines should be folded into one, and moved up right below the 'regular' imports, with a separate `Re-exports` comment header, see https://github.com/Automattic/wp-calypso/blob/master/client/my-sites/themes/controller/index.web.js#L14-L17